akka-stream/src/main/scala/akka/stream/OverflowStrategy.scala

```diff
@@ -55,6 +57,11 @@ object OverflowStrategy {
   def dropBuffer: OverflowStrategy = DropBuffer
 
   /**
+   * If the buffer is full when a new element arrives, drops the new element.
+   */
+  def dropNew: OverflowStrategy = DropNew
```

Comment:
Initiating naming bikeshed :wink: dropNew sounds weird to me, is `dropIncoming` better or weirder?